src/components/elements/dropdown.tsx

```diff
@@ -124,10 +114,10 @@ export const DropdownContainer = styled.ul<DropdownContainerProps>`
     ${(props) => {
         if (props.error)
             return css`
-                border-color: var(--color-red-1) !important;
+                border-color: #ff444f !important;
```

Comment:
@mohammad-hashemi-deriv can you please use this variable `--color-red` insted of color code
Thanks man 🙏